contributing.md

```diff
@@ -0,0 +1,101 @@
+# The git stuff
```

Comment:
Meh, usually we use reStructuredText instead of Markdown... Markdown is trailing whitespace sensitive and thus evil.